File: apps/cargo-tracker/src/main/java/com/example/cargotracker/booking/domain/model/valueobjects/BookingStatus.java
package com.example.cargotracker.booking.domain.model.valueobjects;

public enum BookingStatus {
    PROVISIONAL("仮予約"),
    CONFIRMED("確定済"),
    SETTLED("精算済");

    private final String displayName;

    BookingStatus(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }
}

File: docs/article/source/csharp/apps/cargo-tracker/src/CargoTracker.Web/Booking/Domain/Model/BookingStatus.cs
namespace CargoTracker.Booking.Domain.Model;

/// <summary>予約状態。本 IT では Preliminary 起点と RouteProposed への遷移のみ実装する。</summary>
public enum BookingStatus
{
    Preliminary,
    RouteProposed,
    Confirmed,
    TrackingIssued,
    InTransit,
    Delivered,
    Settled,
    Cancelled,
}
